mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -513,6 +539,7 @@ class SparseMatrix(
    * @param rowIndices the row index of the entry. They must be in strictly increasing
    *                   order for each column
    * @param values non-zero matrix entries in column major
+   * @since 1.3.0
```

Comment:
Is tagging the version for the alternative constructor useful for the end user? If it is, then it should be done for all uniformly.